jina/serve/runtimes/worker/request_handling.py

```diff
@@ -440,15 +439,15 @@ async def handle(
 
     @staticmethod
     def replace_docs(
-        request: List['DataRequest'], docs: 'DocumentArray', ndarrray_type: str = None
+        request: List['DataRequest'],
+        docs: 'DocumentArray',
     ) -> None:
         """Replaces the docs in a message with new Documents.
 
         :param request: The request object
         :param docs: the new docs to be used
-        :param ndarrray_type: type tensor and embedding will be converted to
         """
-        request.data.set_docs_convert_arrays(docs, ndarray_type=ndarrray_type)
```

Comment:
how will this conversion happen?

Reply:
I just disable it for now. Will bring back later for now I am just testing things

Reply:
resolved